docs/usage/install/underlay/get-started-kind-zh_CN.md

```diff
@@ -11,7 +11,7 @@ Kind 是一个使用 Docker 容器节点运行本地 Kubernetes 集群的工具
 ## Kind 上集群部署 Spiderpool
 
 1. 克隆 Spiderpool 代码仓库到本地主机上，并进入 Spiderpool 工程的根目录。
-  
+
```

Comment:
refer to  https://github.com/spidernet-io/spiderpool/blob/main/docs/develop/contributing.md

```
        # for china developer not able access ghcr.io
        # it pulls images from another image registry and just use http proxy to pull chart 
        $ make e2e_init_underlay  -e E2E_CHINA_IMAGE_REGISTRY=true -e HTTP_PROXY=http://${ADDR}
```